wasm.md

```diff
@@ -155,9 +155,10 @@ The sorts `EmptyStmt` and `EmptyStmts` are administrative so that the empty list
     syntax Defns  ::= EmptyStmts
     syntax Stmts  ::= Instrs | Defns
  // --------------------------------
-    rule          <k> .Stmts                => .       ... </k>
-    rule          <k> (S:Stmt .Stmts):KItem => S       ... </k>
-    rule [step] : <k> (S:Stmt SS)           => S ~> SS ... </k> requires SS =/=K .Stmts
+    rule               <k> .Stmts              => .       ... </k>
+    rule [stepStmt]  : <k> (S:Stmt  SS)        => S ~> SS ... </k> requires notBool (isInstr(S) orBool isDefn(S))
+    rule [stepInstr] : <k> (I:Instr IS:Instrs) => I ~> IS ... </k>
+    rule [stepDefn]  : <k> (D:Defn  DS:Defns)  => D ~> DS ... </k>
```

Comment:
I don't dislike this. It's a bit more verbose but less complex than relying on the subsorting into `Stmts`.